test/erc721_test_constants.js

```diff
@@ -21,16 +21,16 @@ const creation_success_types = [
 
 const claim_success_encode = 'ClaimSuccess(bytes32,address,uint256,address)'
 const claim_success_types = [
-  { type: 'bytes32', name: 'id' },
-  { type: 'address', name: 'claimer' },
+  { type: 'bytes32', name: 'id', indexed: true },
+  { type: 'address', name: 'claimer', indexed: true },
   { type: 'uint256', name: 'claimed_token_id' },
   { type: 'address', name: 'token_address' },
 ]
 
 const refund_success_encode = 'RefundSuccess(bytes32,address,uint256,uint256[])'
```

Comment:
I guess we can extract the event type/encode information from the compiled json file(called filters). I am wondering if we can simplify code like this because t he event type/encode information is embedded in the json output. Here is an [example](https://github.com/DimensionDev/InitialTwitterOffering/blob/master/test/TestTP.js#L159)

Reply:
Yes, using filter definitely works but the `getLogs( )` is from the `ethers.js`. In this project, we are testing with Web3.js & Truffle. So we'd better keep the consistency. If we want to migrate to Ethers.js & Waffle, I think it's better to modify the entire testing file.

Reply:
> I guess we can extract the event type/encode information from the compiled json file(called filters). I am wondering if we can simplify code like this because t he event type/encode information is embedded in the json output. Here is an [example](https://github.com/DimensionDev/InitialTwitterOffering/blob/master/test/TestTP.js#L159)

Yes, using filter definitely works but `getLogs()` is from the `Ethers.js`. In this project, we are testing with Web3.js & Truffle.  We'd better keep the consistency. Or If we want to migrate to Ethers.js & Waffle suites, I think it may be better to modify the entire testing file.